docker-compose.yml

```diff
@@ -6,7 +6,7 @@ jekyll:
     - ./:/code:cached
   working_dir: /code
   ports:
-    - "4000:4000"
+    - "5000:4000"
```

Comment:
neee :) ja su na to zvyklej :)

Reply:
ok, ale potom nemozes mat spustene obidva dokumentacie. revertujem ...

Reply:
(bez upravy commandu)

Reply:
ja vim :) ale normalne delas jedno a pak druhy :)